packages/components/src/components/dialog/dialog.stories.ts

```diff
@@ -57,271 +67,73 @@ export const Default = {
-              event.preventDefault();
-            }
-          });
-        </script>
-
-        <sd-dialog id="timed" headline="Timed Dialog" no-close-button>
-          <div id="countdown">Closable in 5 seconds...</div>
-          <sd-button slot="footer" class="w-full" id="timed-close-button" disabled>Close</sd-button>
-        </sd-dialog>
-
-        <script>
-          // Prevent closing the dialog for a certain amount of time
-          const openTimedDialogButton = document.querySelector('#open-timed-dialog');
-          const countdownElement = document.querySelector('#countdown');
-          const timedCloseButton = document.querySelector('#timed-close-button');
-
-          const timedDialog = document.querySelector('#timed');
-          let canCloseTimedDialog = false;
-
-          timedCloseButton.addEventListener('click', () => timedDialog.hide());
-
-          openTimedDialogButton.addEventListener('click', () => {
-            timedCloseButton.disabled = true;
-            timedDialog.show();
-            canCloseTimedDialog = false;
-            let counter = 5;
-            countdownElement.textContent = 'Closable in ' + counter + ' seconds...';
-
-            const interval = setInterval(() => {
-              counter--;
-              countdownElement.textContent = 'Closable in ' + counter + ' seconds...';
-              if (counter <= 0) {
-                clearInterval(interval);
-                canCloseTimedDialog = true;
-                timedCloseButton.disabled = false;
-                countdownElement.textContent = 'You can now close the dialog.';
-              }
-            }, 1000);
-          });
-
-          timedDialog.addEventListener('sd-request-close', event => {
-            if (!canCloseTimedDialog) {
-              event.preventDefault();
-            }
-          });
-        </script>
-      </div>
-    `;
-  }
-};
-
-/**
- * sd-dialog is fully accessibile via keyboard.
- */
-
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless" style="height: 40vh;">${generateTemplate({ args })}</div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-dialog');
-    await waitUntil(() => el?.shadowRoot?.querySelector('sd-button'));
-
-    el?.shadowRoot?.querySelector<HTMLElement>('sd-button')!.focus();
-  }
+export const NoCloseButton = {
+  name: 'No Close Button',
+  render: () => html`
+    <div class="h-[40vh]">
+      <sd-dialog id="dialog" headline="Headline" open no-close-button>
+        <p class="sd-paragraph">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+          dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam.
+        </p>
+        <sd-button class="close-button" slot="footer" variant="primary">Accept Terms and Conditions</sd-button>
```

Comment:
I know there is an ongoing discussion on the content of the button but this needs to be fixed.

Reply:
This has been addressed